wp-content/mu-plugins/rest-collection-params.php

```diff
@@ -23,3 +23,9 @@
 
```

Comment:
This is needed to allow the orderby URL query parameter on the archive page, because it's not a standard WordPress query parameter. See https://developer.wordpress.org/reference/hooks/rest_this-post_type_collection_params/. This file also has similar method calls for the other archive types